hackathon_site/dashboard/frontend/src/components/general/Header/Header.module.scss

```diff
@@ -30,4 +30,27 @@
             background-color: rgba(355, 355, 355, 0.2);
         }
     }
+
+    &Drawer {
+        &Top {
+            width: 100%;
+            height: 56px;
+            display: block;
+            background-color: color(light1);
+            width: 300px;
```

Comment:
you have `width: 100%` at the top.

I don't think you should be specifying the width here either way, in one div and expecting the rest to grow to it. Though for whatever CSS magic reason, I can't set `width: 300px` on `&Drawer` and have it behave.